src/lucky/base_app_server.cr

```diff
@@ -20,8 +20,7 @@ abstract class Lucky::BaseAppServer
 
   # :nodoc:
   def listen : Nil
-    server.bind_tcp host, port
-    server.listen
+    server.listen(host, port)
```

Comment:
Why do we even define this here? We end up overriding it in the generated apps https://github.com/luckyframework/lucky_cli/blob/master/src/web_app_skeleton/src/app_server.cr.ecr#L28-L32 which includes the extra arg for the reuse_port... 🤔  Seems like we could just make this method abstract, then do this cleanup on the CLI repo. What do you think?

Reply:
Good catch, I made the update here and opened https://github.com/luckyframework/lucky_cli/pull/699